### PR TITLE
Add some CSS for the homepage

### PR DESCRIPTION
**Pull request by dalisc**
Changes:
- Put the product cards in tabs
- Display 8 products per page (2 rows of 4 items) though @jacoblipech will be the one implementing the pagination
- Change background color of all pages to grey in global.css

WIP but we can incrementally merge in because I am affecting the js as well so wanna prevent merge conflicts with Jacob.

### CODE REVIEW

File: src/main/webapp/assets/js/index.js [hunk 1]
```diff
@@ -164,13 +196,7 @@ function homePageListingTemplate(business) {
           <p class="categories">${product.categories}</p>
           <p class="price">Price: From \$${product.minPrice}</p>
           <p>${description}...</p>
-          <p class="rating">Rating: ${product.aggregatedRating}</p>
-          <span>
-            <button>
-              <i class="fa fa-cart-arrow-down"></i>
-              <a href=${product.contactUrl}>Contact Business</a>
```
Comment (jacoblipech) [inline src/main/webapp/assets/js/index.js:171]: Why is this part removed tho?

Comment (dalisc) [reply src/main/webapp/assets/js/index.js:171]: i derped and i thought this linked to to the product itself. Now i realised it's the specific contactUrl. I'll put it back in

**Comment by dalisc** [comment]
> New businesses aren't displaying for me. Does it work on your end?

from my branch or develop branch?  jacob just merged something in

my branch is fine for me

anyway im not affecting that code, im just changing the appearance. might wanna talk to jacob about it?

**Comment by dalisc** [comment]
> New businesses aren't displaying for me. Does it work on your end?

### if you have more than 8, only the first 8 will appear! pagination has not been implemented by jacob yet

**Comment by g-lilian** [comment]
> > New businesses aren't displaying for me. Does it work on your end?
> 
> from my branch or develop branch? jacob just merged something in
> 
> my branch is fine for me
> 
> anyway im not affecting that code, im just changing the appearance. might wanna talk to jacob about it?

From feature/css :O It's not just businesses after 8, the first one I add doesn't appear either. So it's working for you? That's weird.

**Comment by jirehcwe** [comment]
Found the error, it was a missing closing bracket :D

**Comment by dalisc** [comment]
Wow thanks Jireh! So can this be approved and merged in?

**Comment by jirehcwe** [comment]
@dalisc CSS wise looks alright to me!
